packages/io/dxf-serializer/README.md

```diff
@@ -1,19 +1,19 @@
 ## @jscad/dxf-serializer
 
-> DXF serializer for the JSCAD project (from CSG)
+> Serializer of CSG geometries to DXF entities
 
 [![npm version](https://badge.fury.io/js/%40jscad%2Fdxf-serializer.svg)](https://badge.fury.io/js/%40jscad%2Fdxf-serializer)
 [![Build Status](https://travis-ci.org/jscad/io.svg)](https://travis-ci.org/jscad/dxf-serializer)
 
 ## Overview
 
-This serializer outputs a 'blobable' array of data (from one or more CSG library objects).
+This serializer outputs a 'blobable' array of data from one or more CSG geometries. Currently, only TEXT output is supported.
```

Comment:
oh, I did not know that only text is supported, good to know (I understand given the complexity!)
Btw perhaps it should perhaps be JSCAD script TEXT output ?

Reply:
Done